packages/tree/tests/tree.browser.spec.ts

```diff
@@ -0,0 +1,449 @@
+                                ]
+                            },
+                            { label: "Operator", size: 2490 },
+                            { label: "OperatorList", size: 5248 },
+                            { label: "OperatorSequence", size: 4190 },
+                            { label: "OperatorSwitch", size: 2581 },
+                            { label: "SortOperator", size: 2023 }
+                        ]
+                    },
+                    { label: "Visualization", size: 16540 }
+                ]
+            }
+        ]
+    }
+};
+
+describe("@hpcc-js/tree", () => {
+    for (const key in tree) {
+        const item = (tree as any)[key];
+        if (item) {
+            if (!urlSearch || urlSearch === item.prototype.constructor.name) {
+                describe(`${item.prototype?.constructor?.name}`, () => {
+                    it("Simple", () => {
+                        expect(true).to.be.true;
+                    });
+                    if (item.prototype instanceof Class) {
+                        classDef("tree", item);
+                    }
+                    if (item.prototype instanceof HTMLWidget || item.prototype instanceof SVGWidget) {
+                        switch (item.prototype.constructor) {
+                            case CirclePacking:
+                                render(new CirclePacking()
+                                    .data(flare.data)
+                                );
+                                break;
+                            case Dendrogram:
+                                render(new Dendrogram()
+                                    .data(flare.data)
+                                );
+                                break;
+                            case DirectoryTree:
+                                render(new DirectoryTree()
+                                    .data(flare.data)
+                                );
+                                break;
+                            case Indented:
+                                render(new Indented()
+                                    .columns(["label", "size"])
+                                    .data(flare.data)
+                                );
+                                break;
+                            case SunburstPartition:
+                                render(new SunburstPartition()
+                                    .data(flare.data)
+                                );
+                                break;
+                            case Treemap:
+                                render(new Treemap()
+                                    .columns(["label", "size"])
+                                    .data(flare.data)
+                                );
+                                break;
+
+                            default:
+                                it("Has render test", () => {
```

Comment:
I believe it's been the case in the last few PRs I reviewed also, but I only just thought about it now. Should these other cases not also have some kind of `it(...)` expectation in them as well?

Reply:
The `it` call is buried inside the "render" function, it does a few checks but the main one is:
1. Set target to a placeholder div
2. Renders the widget
3. Sets the target to null
4. Sets the target to a new placeholder
5. Renders the widget

Reply:
I realized that when I was just reviewing another; chased what that `render(...)` import actually was